internal/commands/waypoint/tfcconfig/tfc_config_read.go

```diff
@@ -67,6 +67,9 @@ func readRun(opts *TFCConfigReadOpts) error {
 	if err != nil {
 		return errors.Wrapf(err, "%s failed to get TFC Config", opts.IO.ColorScheme().FailureIcon())
 	}
+	if resp.Payload.TfcConfig == nil {
+		return errors.New("No TFC Config found for this project")
```

Comment:
Is this an error though? I think given that we updated our API to say no TFC config isn't an error, this simply returns 0.

Reply:
What Brian said; I think we should just output "no tfc config found" or something.

Reply:
Sure we can just output that message without calling it an error.